src/prommis/solid_handling/tests/test_crusher.py

```diff
@@ -0,0 +1,114 @@
+
+# -----------------------------------------------------------------------------
+# Get default solver for testing
+solver = get_solver()
+
+
+# -----------------------------------------------------------------------------
+@pytest.mark.unit
+def test_config():
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(dynamic=False)
+    m.fs.properties_solid = CoalRefuseParameters(
+        doc="solid property",
+    )
+
+    m.fs.unit = CrushAndBreakageUnit(property_package=m.fs.properties_solid)
+
+    # Assert specific config options as per your model's requirements
+    # Example:
+    assert not m.fs.unit.config.dynamic
+    assert not m.fs.unit.config.has_holdup
+    assert m.fs.unit.config.property_package is m.fs.properties_solid
+
+
+# -----------------------------------------------------------------------------
+class TestSolidHandling(object):
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(dynamic=False)
+        m.fs.properties_solid = CoalRefuseParameters(
+            doc="solid property",
+        )
+
+        m.fs.unit = CrushAndBreakageUnit(property_package=m.fs.properties_solid)
+        # Set up your model initialization here
+        m.fs.unit.inlet.mass_frac_comp[0, :].fix(
+            0.1
+        )  # set mass frac value. There are 13 component in property package
+        m.fs.unit.properties_in[0].flow_mass.fix(2000)  # kg/hr
+        m.fs.unit.properties_in[0].particle_size_median.fix(80)  # micrometer
+        m.fs.unit.properties_in[0].particle_size_width.fix(1.5)  # dimensionless
+        m.fs.unit.properties_out[0].particle_size_median.fix(58)  # micrometer
+        m.fs.unit.properties_out[0].particle_size_width.fix(1.5)  # dimensionless
+        return m
+
+    @pytest.mark.build
+    @pytest.mark.unit
+    def test_build(self, model):
+        # Assertions related to the build status of the model
+        # More assertions as needed for your model
+
+        assert number_variables(model.fs.unit) == 59
+        assert number_total_constraints(model.fs.unit) == 41
+        assert number_unused_variables(model.fs.unit) == 0
+
+    @pytest.mark.component
+    def test_structural_issues(self, model):
+        dt = DiagnosticsToolbox(model=model)
+        dt.assert_no_structural_warnings()
+
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_initialize(self, model):
+        initializer = BlockTriangularizationInitializer(constraint_tolerance=2e-5)
+        initializer.initialize(model.fs.unit)
+        assert initializer.summary[model.fs.unit]["status"] == InitializationStatus.Ok
+
+    def test_solve(self, model):
+        results = solver.solve(model)
+        assert_optimal_termination(results)
+
+    def test_solution(self, model):
+        assert pytest.approx(2000.0, rel=1e-5) == value(
+            model.fs.unit.properties_in[0].flow_mass
+        )
```

Comment:
There is little point in testing the feed conditions, as those are fixed (same for the product particle size and width). What you need to check are all the things that are calculated by the unit model; i.e. the outlet flow rate and composition, the P80 values and the work required.